two-fer/two_fer.go

```diff
@@ -0,0 +1,20 @@
+// This is a "stub" file.  It's a little start on your solution.
+// It's not a complete solution though; you have to write some code.
+
+// Package twofer should have a package comment that summarizes what it's about.
+// https://golang.org/doc/effective_go.html#commentary
+package twofer
+
+import "fmt"
+
+// Returns the following formatted string depending on the entryString parameter
+// "One for <entryString>, one for me."
+// parameters:
+// * entryString : string to use in the formatting, replaced by "you" if a
+// zero-length string is passed
+func ShareWith(entryString string) string {
+	if entryString == "" {
+		entryString = "you"
+	}
+	return fmt.Sprintf("One for %s, one for me.", entryString)
```

Comment:
I think it's better to extract the first argument in a constant (called for example `template`) as in the general case it could be reused :slightly_smiling_face: